lighthouse-core/test/audits/metrics-test.js

```diff
@@ -102,19 +105,19 @@ describe('Performance: metrics', () => {
   it('evaluates valid input (with CLS from all frames) correctly', async () => {
     const artifacts = {
       traces: {
-        [MetricsAudit.DEFAULT_PASS]: metricsAllFramesTrace,
+        [MetricsAudit.DEFAULT_PASS]: clsAllFramesTrace,
       },
       devtoolsLogs: {
-        [MetricsAudit.DEFAULT_PASS]: metricsAllFramesDevtoolsLog,
+        [MetricsAudit.DEFAULT_PASS]: clsAllFramesDevtoolsLog,
       },
     };
 
-    const context = {settings: {throttlingMethod: 'simulate'}, computedCache: new Map()};
+    const context = {settings: {throttlingMethod: 'provided'}, computedCache: new Map()};
```

Comment:
was there significance to the flip from provided to simulate?

Reply:
No, I think this was an existing mistake. The trace is from a DT throttled run.

Since it's CLS, this doesn't actually affect anything though, it's more of an annotation.